source/admin_manual/list_of_events.rst

```diff
@@ -854,579 +482,384 @@ These events apply only for connections using the ``connection API``.
-| Field               | Description                                          |
-+=====================+======================================================+
-| bytes_in            | Amount of data read, in bytes                        |
-+---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
-+---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
-+---------------------+------------------------------------------------------+
-
-
-server_connection_connected
----------------------------
-
-Emitted when an outgoing server connection was either successfully established
-or failed. Currently it is not possible to know which one happened.
-
-server_connection_disconnected
-------------------------------
-
-Emitted when a server connection is terminated.
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| bytes_in            | Amount of data read, in bytes                        |
-+---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
-+---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
-+---------------------+------------------------------------------------------+
 
 FS
 ==
 
-.. _event_fs:
+.. dovecot_core:event:: fs
 
-fs
---
+   Inherits from environment (e.g. :mail_user event)
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from environment (e.g. :ref:`event_mail_user`)                    |
-+---------------------+------------------------------------------------------+
+   .. todo:: WHAT DOES THIS MEAN???
 
-.. _event_fs_file:
 
-fs_file
--------
+.. dovecot_core:event:: fs_file
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from :ref:`event_fs` or any other specified event                 |
-| (e.g. :ref:`event_mail`)                                                   |
-+---------------------+------------------------------------------------------+
+   Inherits from fs or any other specified event (e.g. mail).
 
-.. _event_fs_iter:
+   .. todo:: WHAT DOES THIS MEAN???
```

Comment:
Pretty much what it says. The parent event comes from a parameter, so it can be anything.